src/rpc/rawtransaction_util.cpp

```diff
@@ -18,7 +18,7 @@
 #include <util/rbf.h>
 #include <util/strencodings.h>
 
-CMutableTransaction ConstructTransaction(const UniValue& inputs_in, const UniValue& outputs_in, const UniValue& locktime, const UniValue& rbf)
+CMutableTransaction ConstructTransaction(const UniValue& inputs_in, const UniValue& outputs_in, const UniValue& locktime, bool rbf)
```

Comment:
Will `ConstructTransaction` be moved out of the RPC module?

Without this change the issue could probably have been addressed with a small update in `walletcreatefundedpsbt`.

If `ConstructTransaction` stays in the RPC codebase forever then this change has also introduced duplicate code at two call sites for no reason.

Reply:
I expect more stuff to be moved out of the RPC over time, but for now this was the simplest fix, also easiest to backport.